lib/tasks/data_migrations/create_shared_footer_properties.rb.rake

```diff
@@ -0,0 +1,33 @@
+namespace :data_migrations do
+  desc 'Creates settings to fetch shared footer from VOOG site'
+  task create_shared_footer_fetcher_settings: :environment do
+    hash = {
+      voog_site_url: {
+        description: <<~TEXT.squish,
+          VOOG site from which to fetch localized footer elements. Defaults to https://www.internet.ee.
+        TEXT
+        value: 'https://www.internet.ee',
```

Comment:
Maybe we should use the new Setting structure with formats?

Reply:
Totally missed this one. Thanks!